lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -514,6 +514,14 @@ public void OnNext(IFailedEvaluator failedEvaluator)
                         return;
                     }
 
+                    if (_taskManager != null && _taskManager.IsMasterTaskCompleted())
+                    {
+                        Logger.Log(Level.Info,
+                           "Master task have been completed running. So ignoring the Evaluator {0} failure and execute DoneAction.", failedEvaluator.Id);
```

Comment:
`Master task has completed successfully. Ignoring Evaluator {0} failure and completing the job.`